src/components/@shared/AvailableNetworks/networkOptions.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from 'react'
+import NetworkName from '@shared/NetworkName'
+import styles from './NetworkOptions.module.css'
+import Button from '@shared/atoms/Button'
+import useNetworkMetadata from '@hooks/useNetworkMetadata'
+import { addCustomNetwork } from '@utils/web3'
+import { useWeb3 } from '@context/Web3'
+
+export default function NetworkOptions({
+  chainId
+}: {
+  chainId: number
+}): ReactElement {
+  const { networksList } = useNetworkMetadata()
+  const { web3Provider } = useWeb3()
+
+  function changeNetwork(chainId: number) {
+    const networkNode = networksList.find((data) => data.chainId === chainId)
+    addCustomNetwork(web3Provider, networkNode)
+  }
+
+  return (
+    <div key={chainId}>
```

Comment:
that's a weird markup for a list, and gets even more wrapping divs when rendered with `AvailableNetworks/index.tsx`. Semantically, it's a list and not a form so no form elements like `label`. Should be a `ul` with `button` inside of `li` and that's it, no further divs